stl/inc/ranges

```diff
@@ -872,6 +872,133 @@ namespace ranges {
 
         inline constexpr _Reverse_fn reverse;
     } // namespace views
+
+    // CLASS TEMPLATE ranges::common_view
+    // clang-format off
+    template <view _Vw>
+        requires (!common_range<_Vw> && copyable<iterator_t<_Vw>>)
+    class common_view : public view_interface<common_view<_Vw>> {
+        // clang-format on
+    private:
+        /* [[no_unique_address]] */ _Vw _Base{};
+
+    public:
+        common_view() = default;
+        constexpr explicit common_view(_Vw _Base_) noexcept(is_nothrow_move_constructible_v<_Vw>) // strengthened
+            : _Base(_STD move(_Base_)) {}
+
+        // clang-format off
+        template <viewable_range _Rng>
+            requires (!common_range<_Rng> && constructible_from<_Vw, views::all_t<_Rng>>)
+        constexpr explicit common_view(_Rng&& _Range) noexcept(

```

Comment:
This constructor is no more, per LWG-3405. Maybe leave a comment `// converting constructor template omitted per LWG-3405` since it hasn't been applied to the working draft yet.

Reply:
You mean the standard betrayed me?

Reply:
Done